src/en/guide/REST/callingRestServices.adoc

```diff
@@ -0,0 +1,76 @@
+Calling Grails REST services - as well as third-party services - is very straightforward using the https://docs.micronaut.io/latest/guide/index.html#httpClient[Micronaut HTTP Client]. This HTTP client has both a low-level API and a higher level AOP-driven API, making it useful for both simple requests as well as building declarative, type-safe API layers.
+
+To use the Micronaut HTTP client you must have the `micronaut-http-client` dependency on your classpath. Add the following dependency to your `build.gradle` file.
+
+.build.gradle
+[source,groovy]
+----
+compile 'io.micronaut:micronaut-http-client'
+----
+
+==== Low-level API
+
+The https://docs.micronaut.io/latest/api/io/micronaut/http/client/HttpClient.html[HttpClient] interface forms the basis for the low-level API. This interfaces declares methods to help ease executing HTTP requests and receive responses.
+
+The majority of the methods in the `HttpClient` interface returns Reactive Streams Publisher instances, and a sub-interface called RxHttpClient is included that provides a variation of the HttpClient interface that returns RxJava Flowable types. When using `HttpClient` in a blocking flow, you may wish to call `toBlocking()` to return an instance of https://docs.micronaut.io/latest/api/io/micronaut/http/client/BlockingHttpClient.html[BlockingHttpClient].
+
+There are a few ways by which you can obtain a reference to a https://docs.micronaut.io/latest/api/io/micronaut/http/client/HttpClient.html[HttpClient]. The most simple way is using the https://docs.micronaut.io/latest/api/io/micronaut/http/client/HttpClient.html#create-java.net.URL-[create] method
+
+.Creating an HTTP client
+[source, groovy]
+----
+    List<Album> searchWithApi(String searchTerm) {
+        String baseUrl = "https://itunes.apple.com/"
+
+        HttpClient client = HttpClient.create(baseUrl.toURL()).toBlocking() //<1>
```

Comment:
The client should be defined as a spring bean and shutdown with the `close` method. Not closing the client will leak threads.